[caffe2][be] migrate global static initializer - version_map

klein-meta opened the pull request:
Summary: Caffe2 lib has 200+ global static initializer usage, which are papar-cut reference to startup perf. Detail in this post https://fb.workplace.com/groups/arglassesperf/permalink/623909116287154.

Test Plan: CI

Differential Revision: D58643791

aaronenyeshi reviewed (approved):
Please fix the CI issues, may be occuring on another diff lower in the stack

klein-meta committed:
[caffe2][be] migrate global static initializer - version_map (#128831)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/128831

Caffe2 lib has 200+ global static initializer usage, which are papar-cut reference to startup perf. Detail in this post https://fb.workplace.com/groups/arglassesperf/permalink/623909116287154.

Test Plan: CI

Reviewed By: aaronenyeshi

Differential Revision: D58643791
